app/_locales/en/messages.json

```diff
@@ -1565,6 +1565,27 @@
   "history": {
     "message": "History"
   },
+  "holdToReveal": {
+    "message": "Hold to reveal SRP"
+  },
+  "holdToRevealContent1": {
+    "message": "Your Secret Recovery Phrase provides $1"
```

Comment:
Could you provide a `description` field for these messages? We use the `description` field to provide context to our translators about what this message means and the context in which it's used.

It's especially important for messages that include parameters, that we explain what the parameter is. It can affect the grammar in unexpected ways.